optaplanner-website-root/content/blog/2023/04/26/BoostVRPListSelectorScalability.adoc

```diff
@@ -0,0 +1,102 @@
+= Scaling Up Vehicle Routing Problem with planning list variable and Nearby Selector
+dupliaka
+2023-04-26
```

Comment:
Please update again (together with directories) either to this day or tomorrow.

Reply:
Updated 6510f062